metadata_xml/validation.py

```diff
@@ -183,6 +249,19 @@ def validate(record):
     if fields_with_bad_dates:
         errors.append("""Date/time formatting error in field(s): {}.
                                  Pattern must match one of: {}""".format(
-            join(fields_with_bad_dates), join(acceptable_date_formats_text)))
+            join(fields_with_bad_dates), join(acceptable_date_formats)))
```

Comment:
Minor but it seems somewhat confusing to create a function join that calls str.join - night be clearer to use a slightly different name? Or define e.g. comma = ', ' and then use comma.join(lst).

Reply:
I like the `comma.join()` idea